src/main/kotlin/com/londogard/summarize/summarizers/EmbeddingClusterSummarizer.kt

```diff
@@ -13,11 +13,11 @@ enum class ScoringConfig {
     Ghalandari
 }
 
-class EmbeddingClusterSummarizer(
+internal class EmbeddingClusterSummarizer(
     val threshold: Double = 0.3,
     val simThreshold: Double = 0.95,
     val config: ScoringConfig = ScoringConfig.Ghalandari
-) : SmileOperators, Summarizer {
+) : SmileOperators, SummarizerModel {
     private val embeddings: WordEmbeddings = WordEmbeddings(dimensions = 50)
     private val zeroArray = Array(embeddings.dimensions) { 0f }
     private fun String.simplify(): String =
```

Comment:
Något jag inte ser eller slutar raden med "=" ?

Reply:
Something you're not seeing. Making it into a one-liner